includes/basics/baseurl.md

```diff
@@ -39,8 +39,13 @@ Within this address, you'll find APIs (collections of related endpoints) coverin
         <td><code class="language-http">{{ stache.config.gift_api_suffix }}</code></td>
         <td>Used to manage gift information, including related entities such as acknowledgements, fundraisers, and receipts.</td>
       </tr>
+      <tr>
+        <td class="nowrap">Peer-to-Peer (Beta)</td>
+        <td><code class="language-http">{{ stache.config.peer_to_peer_api_suffix }}</code></td>
+        <td>Used to create supporter pages, and query campaigns, donors, donations, and teams.</td>
```

Comment:
Good call. For the previous file, I compared what we had on the website. For this one, I didn't do that. My bad. ... In that case, I'd say leave it as is in your file, and I'll make a note to go back and update all of the instances at once when I get a chance. ... Sorry about that.